<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projekt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>70</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="sldD">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>410</y>
       <width>0</width>
       <height>0</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>70</y>
      <width>140</width>
      <height>512</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="g1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>25</y>
       <width>120</width>
       <height>95</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSlider" name="przesuniecieX">
        <property name="minimum">
         <number>-200</number>
        </property>
        <property name="maximum">
         <number>700</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="przesuniecieY">
        <property name="minimum">
         <number>-200</number>
        </property>
        <property name="maximum">
         <number>700</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="przesuniecieZ">
        <property name="minimum">
         <number>-200</number>
        </property>
        <property name="maximum">
         <number>700</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="Skalowanie">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>125</y>
       <width>140</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Skalowanie:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="g2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>120</width>
       <height>95</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSlider" name="skalowanieX">
        <property name="maximum">
         <number>3000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="skalowanieY">
        <property name="maximum">
         <number>3000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="skalowanieZ">
        <property name="maximum">
         <number>3000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="g6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>18</width>
       <height>90</height>
      </rect>
     </property>
     <widget class="QLabel" name="lblX_8">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblY_8">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>36</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblZ_8">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>70</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Z:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="g8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>410</y>
       <width>18</width>
       <height>90</height>
      </rect>
     </property>
     <widget class="QLabel" name="lblX_9">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblY_9">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>36</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblZ_9">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>70</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Z:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="g7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>18</width>
       <height>90</height>
      </rect>
     </property>
     <widget class="QLabel" name="lblX_12">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblY_12">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>36</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblZ_12">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>70</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Z:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="Przesuniecie">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>140</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Przesunięcie:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rotacja">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>255</y>
       <width>140</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rotacja:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="g3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>120</width>
       <height>95</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSlider" name="rotacjaX">
        <property name="maximum">
         <number>720</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="rotacjaY">
        <property name="maximum">
         <number>720</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="rotacjaZ">
        <property name="maximum">
         <number>720</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="Pochylenie">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>385</y>
       <width>140</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pochylenie:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="g4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>120</width>
       <height>95</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <widget class="QSlider" name="pochylenieX">
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="pochylenieY">
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="pochylenieZ">
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="g5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>25</y>
       <width>18</width>
       <height>90</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <widget class="QLabel" name="lblX">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblY">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>36</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblZ">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>70</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Z:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox1">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>664</width>
      <height>40</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="reset">
     <property name="geometry">
      <rect>
       <x>444</x>
       <y>0</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #7AC5CD; border-radius: 6px;</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ukryjSciany">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ukryj Ściany</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="teksturowanie">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Teksturowanie</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quit">
     <property name="geometry">
      <rect>
       <x>564</x>
       <y>0</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #7AC5CD; border-radius: 6px;</string>
     </property>
     <property name="text">
      <string>Wyjście</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
